examples/values-sigstore-openshift.yaml

```diff
@@ -1,259 +1,46 @@
+# With this example, it is expected that there is a secret with the fulcio root & signing keys
```

Comment:
The following needs to be added to overriding examples. Otherwise, the contents of `fulcio-secret-rh` is overwritten

```
scaffold:
  fulcio:
    createcerts:
      enabled: false
```

Reply:
added, thanks